README.md

```diff
@@ -250,10 +252,8 @@ export default NuxtAuthHandler({
         } else {
           // eslint-disable-next-line no-console
           console.error('Warning: Malicious login attempt registered, bad credentials provided')
-
           // If you return null then an error will be displayed advising the user to check their details.
           return null
-
```

Comment:
please revert these changes -> out of scope for this PR + like this on purpose to exactly match the code in `[...].ts`